tt_tool.py

```diff
@@ -164,6 +177,12 @@
     if args.harden:
         project.harden()
 
+    if args.run_drc:
+        project.run_drc()
+
+    if args.run_lvs:
+        project.run_lvs()
+
```

Comment:
Why not make these part of harden? this way, we have a similar workflow between OL2 and ORFS, and user who follow the local hardening instructions don't have to worry about remembering to run LVS/DRC.